### PR TITLE
Floating images design updates

### PR DESCRIPTION
**Pull request by schooolman**
Updating the styles for the floating-image block to match the new site design. Have also updated primary, secondary, and tertiary colors in the root styles.

Test URLs:
- Before: https://main--hsf-commonmoves--hlxsites.hlx.page/
- After: https://floating-images-design-updates--hsf-commonmoves--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/floating-images/floating-images.css [hunk 1]
```diff
@@ -43,14 +55,32 @@
   line-height: var(--line-height-m);
 }
 
-.block.floating-images p.button-container {
-  text-align: center;
+.block.floating-images p .button {
```
Comment (bstopp) [inline blocks/floating-images/floating-images.css:58]: Can you check this and compare it to the use of `p.button-container a` and/or `p.button-container a.primary` throughout the site, if we can just update those, that'd be better.

Comment (schooolman) [reply blocks/floating-images/floating-images.css]: Took a look and it seems like this is the main primary button style of the new design, so I moved those button styles to the main styles.css file.

File: styles/styles.css [hunk 2]
```diff
@@ -368,20 +370,32 @@ main .section .default-content-wrapper picture img {
 }
 
 .button-container a {
+  background-color: transparent;
+  border: 1px solid var(--primary-color);
+  color: var(--primary-color);
+  cursor: pointer;
   display: inline-block;
-  padding: .75em 1.5em;
   font-family: var(--font-family-proxima);
```
Comment (bstopp) [inline styles/styles.css:378]: Is this still Proxima?

**Review by bstopp — CHANGES_REQUESTED** [review]

File: styles/styles.css (hunk 2 above)
Comment (schooolman) [reply styles/styles.css:378]: I was planning on updating the fonts in a separate pull request, and then updating all `--font-family-proxima` to use the new font, and changing the variable name to `--font-family-primary`

I can do that in this PR if you would prefer though.

**Review by bstopp — APPROVED** [review]

**Pull request merged**